core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -188,61 +240,34 @@ jit_compile_op_call_draft(JitCompContext *cc, uint32 func_idx, bool tail_call)
         GEN_INSN(LDPTR, native_func, import_func_ptrs,
                  NEW_CONST(I32, (uint32)sizeof(void *) * func_idx));
 
-        /* Initialize parameter types */
-        total_size = sizeof(uint8) * (uint64)(func_type->param_count + 1);
+        /* allocate memory for argvs*/
+        total_size = sizeof(JitReg) * (uint64)(func_type->param_count);
         if (total_size >= UINT32_MAX
-            || !(param_types = wasm_runtime_malloc((uint32)total_size))) {
-            /* exception */
+            || !(argvs = wasm_runtime_malloc((uint32)total_size))) {
```

Comment:
Use jit_malloc or jit_calloc, in fast-jit we usually use them.

Reply:
Should jit_free(argvs) before function return: before return true or return false.